virttest/utils_test/qemu.py

```diff
@@ -1460,9 +1460,11 @@ def action_after_suspend(self, **args):
         pass
 
 
-class MemoryHotplugTest(object):
+class MemoryComm(object):
 
-    UNIT = "M"
+    """
```

Comment:
Hi @xutian 

I have not catch your meaning, could you explain it?

Btw, others have been updated, thanks for the comments.

Thanks.